### DISCUSSION TITLE
Uniswap V3. How to remove liquidity.

### DISCUSSION THREAD

**Discussion by nadtech-hub** [Uniswap/interface#2717] [Q&A]
I'm trying follow sdk guide about removing liquidity. 
https://docs.uniswap.org/sdk/guides/liquidity/removing
What function to execute with given calldata and value?

**Comment by apancorb** [comment]
have you been able to find a solution for this?

**Reply by mostrovoi** [reply]
Just send a transaction with the calldata to uniswap v3 position nft contract

**Reply by pancakeswap-Uniswap** [reply]
Hi, there was the same error at the moment, I found an up-to-date solution to the problem - https://medium.com/@argentosalick/confirm-not-working-uniswap-be866d4d6400
